roles/kubernetes/control-plane/tasks/kubeadm-secondary.yml

```diff
@@ -19,6 +20,7 @@
   register: kubeadm_upload_cert
   when:
     - inventory_hostname == first_kube_control_plane
+    - not kube_external_ca_mode
```

Comment:
Where is `kube_external_ca_mode` defined? It should have a sane default and be documented so that deployers know about this functionality and the deployment doesn't fail if the value is not defined.